packages/gatsby-design-tokens/src/z-indices.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  feedbackWidget: 2,
```

Comment:
maybe 
```
widget: 2
```
it could be used for other widgets in the future too

Reply:
Renamed in 273bb2e

I'm not super sure that all widgets will work with a z-index of 2 though.
Want to get together and work through some solid definitions for "depth/elevation" (`z-index`)?

Reply:
the shorter version is just less specific, but sure it could happen that it will not work in all cases. But you are right, maybe we should talk about something truly general like `level0`, 'level1`

Reply:
👍

Reply:
Just realized I messed up the renaming of the two `zIndices` things—did them in `utils/tokens/z-indices` … 🤦‍♂ https://github.com/gatsbyjs/gatsby/commit/dded37dd028ebb311dbe6930b7b2633148fcf612#diff-d3cc05df2a27981e190be3123cae8e92 … ah well. `0.0.2` incoming.